### DISCUSSION TITLE
Challenges in Mapping Lookup Function Results

### DISCUSSION THREAD

**Discussion by asadleo94** [cloud-carbon-footprint/cloud-carbon-footprint#1268] [Q&A]
Hi,

I need your support. I am attempting to calculate the carbon footprint of AWS S3 based on usage through the lookup function. I have tried it, and it is providing results in terms of CO2e and KWH. However, I am having trouble mapping these results to my storage usage. For example, some S3 buckets have 100GB of data, and others have more or less. While I understand that EC2 instances provide hourly KWH, which I can multiply with actual usage, I am a bit confused about the output result from the lookup function of S3, Lambda, or other services and what it actually represents.

Thanks for your support and help.

**Comment by ccasher** [comment]
Hi @asadleo94, sorry for the delayed response. The main idea of the lookup table is to allow you to see a single unit of usage (in this case storage allocated), so to get the output, you need to multiply the CO2e usage by the 100GB of data. Hope that makes sense.
